tests/Tests/500__jupyterhub/pvc-change_verification.robot

```diff
@@ -0,0 +1,76 @@
+*** Settings ***
+Documentation    PVC_CHANGE_VERIFICATION
+...                 Verify  if we can modify default PVC size
+...
+...
+...                 = Variables =
+...                 | Namespace | Required |    Jupyterhub configuration namespace|
+...                 | S_size    | Required |    Supported size |
+...                 | Ns_size   | Required  |   Not Supported PVC size |
+...                 | Size_code | Required |    Python code to check size in notebook |
+#
+Library         SeleniumLibrary
+Library         JupyterLibrary
+Resource         ../../Resources/ODS.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+
+Test Teardown   PVC Size Test Teardown
+
+
+*** Variables ***
+${NAMESPACE}    redhat-ods-applications
+${S_SIZE}       15
+${SIZE_CODE}    import subprocess;
+...    int(subprocess.check_output(['df','-h', '/opt/app-root/src']).split()[8].decode('utf-8')[:-1])
+@{NS_SIZE}      0    -15    6.2    abc
+
+
+*** Test Cases ***
+Verify User Can Spawn Notebook With PVC Change
+    [Documentation]   Verify if user can spawn notebook
+    ...    for supported PVC size git change
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-1228    ODS-1221
+    ...       Resources-PVC
+    Change PVC And Rollout JH Deployment Config    ${S_SIZE}Gi
```

Comment:
why putting in the same keyword `Change PVC` and `Rollout JH`?

Reply:
It is a custom keyword whihc does both the work. Do you think it is good idea to have separate keyword

Reply:
hm at a first glance, I thought there was no point to put them together, but I think you did it because in order to apply the change you must rollout...yeah maybe an alternative name to make the TC transparent to that would have been something lik `Apply New PVC Size` or `Change and Apply PVC size`. But I think you could leave as it is..

Reply:
done